test/cmd/policy.sh

```diff
@@ -203,7 +203,10 @@ os::cmd::expect_failure_and_text 'oc policy scc-subject-review -z system:service
 os::cmd::expect_failure_and_text 'oc policy scc-review -f ${OS_ROOT}/test/testdata/pspreview_unsupported_statefulset.yaml' 'error: StatefulSet "rd" with spec.volumeClaimTemplates currently not supported.'
 os::cmd::expect_success_and_text 'oc policy scc-subject-review -f ${OS_ROOT}/test/testdata/job.yaml -o=jsonpath={.status.AllowedBy.name}' 'anyuid'
 os::cmd::expect_success_and_text 'oc policy scc-subject-review -f ${OS_ROOT}/test/testdata/redis-slave.yaml -o=jsonpath={.status.AllowedBy.name}' 'anyuid'
-os::cmd::expect_success_and_text 'oc policy scc-subject-review -f ${OS_ROOT}/test/testdata/nginx_pod.yaml -o=jsonpath={.status.AllowedBy.name}' 'privileged'
+# In the past system:admin only had access to a few SCCs, so the following command resulted in the privileged SCC being used
+# Since SCCs are now authorized via RBAC, and system:admin can perform all RBAC actions == system:admin can access all SCCs now
+# Thus the following command now results in the use of the hostnetwork SCC which is the most restrictive SCC that still allows the pod to run
+os::cmd::expect_success_and_text 'oc policy scc-subject-review -f ${OS_ROOT}/test/testdata/nginx_pod.yaml -o=jsonpath={.status.AllowedBy.name}' 'hostnetwork'
```

Comment:
Huh, that's a bit unexpected but I like it!

Reply:
By the way, looks like it could break some configuration. Do we need to mention in the release notes?

Reply:
How would it break something?  It will stick pick a SCC that allows the pod to run.

Reply:
The same way it broke our tests, it could break others. In fact, now pod got less privileges than it had before. Yes, it's rare case.